panic running influx_inspect info

Running ```influx_inspect info```

gives me 
```
influx_inspect info
[cacheloader] 2016/04/12 23:34:56 reading file /Users/seif/.influxdb/wal/_internal/monitor/1/_00001.wal, size 1077
[cacheloader] 2016/04/12 23:34:56 reading file /Users/seif/.influxdb/wal/_internal/monitor/1/_00002.wal, size 169812
[filestore] 2016/04/12 23:34:56 /Users/seif/.influxdb/data/stress/default/2/000000001-000000001.tsm (#0) opened in 7.417241ms
[cacheloader] 2016/04/12 23:34:56 reading file /Users/seif/.influxdb/wal/stress/default/2/_00006.wal, size 0
[cacheloader] 2016/04/12 23:34:56 reading file /Users/seif/.influxdb/wal/_internal/monitor/1/_00003.wal, size 43295
[cacheloader] 2016/04/12 23:34:56 reading file /Users/seif/.influxdb/wal/_internal/monitor/1/_00006.wal, size 0
[tsm1] 2016/04/12 23:34:56 /Users/seif/.influxdb/data/_internal/monitor/1 database index loaded in 556.748µs
[tsm1] 2016/04/12 23:34:57 /Users/seif/.influxdb/data/stress/default/2 database index loaded in 535.600122ms
Shards: 2, Indexes: 2, Databases: 2, Disk Size: 170, Series: 0

panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0xb8c51]

goroutine 1 [running]:
panic(0x52f420, 0xc8200120e0)
	/usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:464 +0x3e6
github.com/influxdata/influxdb/tsdb.(*FieldCodec).Fields(0x0, 0x0, 0x0, 0x0)
	/Users/seif/Projects/go/src/github.com/influxdata/influxdb/tsdb/shard.go:846 +0x41
main.cmdInfo(0xc8200dc860, 0x15)
	/Users/seif/Projects/go/src/github.com/influxdata/influxdb/cmd/influx_inspect/info.go:65 +0x1019
main.main()
	/Users/seif/Projects/go/src/github.com/influxdata/influxdb/cmd/influx_inspect/main.go:51 +0x56d
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -22,6 +22,7 @@
 - [#6252](https://github.com/influxdata/influxdb/pull/6252): Remove TSDB listener accept message @simnv
 - [#6202](https://github.com/influxdata/influxdb/pull/6202): Check default SHARD DURATION when recreating the same database.
 - [#6296](https://github.com/influxdata/influxdb/issues/6296): Allow the implicit time field to be renamed again.
+- [#6294](https://github.com/influxdata/influxdb/issues/6294): Fix panic running influx_inspect info.
 
 ## v0.12.0 [2016-04-05]
 ### Release Notes

--- a/cmd/influx_inspect/info.go
+++ b/cmd/influx_inspect/info.go
@@ -62,6 +62,9 @@ func cmdInfo(path string) {
 			for _, shardID := range shardIDs {
 				shard := tstore.Shard(shardID)
 				codec := shard.FieldCodec(m.Name)
+				if codec == nil {
+					continue
+				}
 				for _, field := range codec.Fields() {
 					ft := fmt.Sprintf("%s:%s", field.Name, field.Type)
 					fmt.Fprintf(tw, "%d\t%s\t%s\t%d/%d\t%d [%s]\t%d\n", shardID, db, m.Name, len(tags), tagValues,

